Builds failed for ESP8266

Hi Felix
First of all, thank you for your good work, sharing and maintaining it as a free & open-source project.


After 2.0.3 release builds for esp8266 fail because `RTC_DATA_ATTR` is not exists for this platform.

Build log:
```
.pio/libdeps/env_name/ESP32Time/ESP32Time.cpp:29:1: error: 'RTC_DATA_ATTR' does not name a type
   29 | RTC_DATA_ATTR static bool overflow;
      | ^~~~~~~~~~~~~
.pio/libdeps/env_name/ESP32Time/ESP32Time.cpp: In member function 'void ESP32Time::setTime(long unsigned int, int)':
.pio/libdeps/env_name/ESP32Time/ESP32Time.cpp:97:4: error: 'overflow' was not declared in this scope
   97 |    overflow = true;
      |    ^~~~~~~~
.pio/libdeps/env_name/ESP32Time/ESP32Time.cpp:100:4: error: 'overflow' was not declared in this scope
  100 |    overflow = false;
      |    ^~~~~~~~
.pio/libdeps/env_name/ESP32Time/ESP32Time.cpp: In member function 'tm ESP32Time::getTimeStruct()':
.pio/libdeps/env_name/ESP32Time/ESP32Time.cpp:117:7: error: 'overflow' was not declared in this scope
  117 |   if (overflow){
      |       ^~~~~~~~
.pio/libdeps/env_name/ESP32Time/ESP32Time.cpp:126:7: error: 'overflow' was not declared in this scope
  126 |   if (overflow){
      |       ^~~~~~~~
.pio/libdeps/env_name/ESP32Time/ESP32Time.cpp: In member function 'long unsigned int ESP32Time::getLocalEpoch()':
.pio/libdeps/env_name/ESP32Time/ESP32Time.cpp:250:6: error: 'overflow' was not declared in this scope
  250 |  if (overflow){
      |      ^~~~~~~~
*** [.pio/build/env_name/lib911/ESP32Time/ESP32Time.cpp.o] Error 1
```

--- a/ESP32Time.cpp
+++ b/ESP32Time.cpp
@@ -26,7 +26,12 @@
 #include "time.h"
 #include <sys/time.h>
 
+#ifdef RTC_DATA_ATTR
 RTC_DATA_ATTR static bool overflow;
+#else
+static bool overflow;
+#endif
+
 
 /*!
     @brief  Constructor for ESP32Time

